load_routes.php

```diff
@@ -50,5 +50,11 @@
 
 }
 catch (ResourceNotFoundException $e) {
-  echo $e->getMessage();
+  $code = $e->getCode();
+  if ($code == 0) {
+    header('Location: /page-not-found');
```

Comment:
- Remove the `/` (black-slash) from this command because it is causing an error displaying Apache's not found error template.
- Also the route is not correct.
- Removing the `/` solves the issue.

- Screenshot of the error below
Error Screenshot : 
![Screenshot from 2022-02-18 12-30-46](https://user-images.githubusercontent.com/83503327/154633797-59effda4-9775-4fd4-ad46-e5b699433675.png)

Reply:
Done.